### DISCUSSION TITLE
OKX denied the webhook signal. Is it a problem with Pipedream’s IP?

### DISCUSSION THREAD

**Discussion by hebe061103** [soranoo/TradingView-Free-Webhook-Alerts#68] [General]
![截图 2024-03-18 14-14-54](https://github.com/soranoo/TradingView-Free-Webhook-Alerts/assets/24790402/2d72bfe0-4a39-4d1f-ba1a-313884a83ead)

**Comment by soranoo** [comment]
Yes, seems pipedreams server IP got banned.

**Reply by hebe061103** [reply]
知道了,谢谢!有没有其它办法?使用你的本地版,不知道可不可以

**Comment by soranoo** [comment]
Check out https://github.com/soranoo/TradingView-Free-Webhook-Alerts/issues/69 for the update on the solution.
